conf/nginx.conf

```diff
@@ -1,12 +1,7 @@
 location __PATH__/ {
 
-	# Force usage of https
-	if ($scheme = http) {
-		rewrite ^ https://$server_name$request_uri? permanent;
-	}
-
-#sub_path_only	rewrite                ^__PATH__$ __PATH__/ permanent;
-#sub_path_only	rewrite                ^__PATH__/admin$ __PATH__/admin/ permanent;
+	#sub_path_only	rewrite                ^__PATH__$ __PATH__/ permanent;
```

Comment:
Not a regression from this PR but uh, shouldn't this line be *outside* of the location block ? o_O

as in https://github.com/YunoHost/example_ynh/blob/master/conf/nginx.conf#L1

(Otherwise it never matches since `location __PATH__/` already includes the trailing slash)